credentialdigger/client_sqlite.py

```diff
@@ -49,6 +51,14 @@ def __init__(self, path):
                 FOREIGN KEY (rule_id) REFERENCES rules ON DELETE SET NULL ON UPDATE CASCADE
             );
 
+            CREATE TABLE IF NOT EXISTS embeddings (
+                id INTEGER REFERENCES discoveries,
+                snippet TEXT DEFAULT '',
+                embedding TEXT DEFAULT '',
```

Comment:
Do we need a default=''? If I remember correctly, NULL values are allowed in the db (unless you define NOT NULL, which is not the case here)

Reply:
no I tried without, it works as well